client/src/main/java/org/eclipse/hono/client/impl/HonoClientImpl.java

```diff
@@ -12,20 +12,14 @@
  */
 package org.eclipse.hono.client.impl;
 
-import java.net.HttpURLConnection;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.function.BiConsumer;
-import java.util.function.Consumer;
-import java.util.function.Supplier;
-
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.proton.ProtonClientOptions;
+import io.vertx.proton.ProtonConnection;
+import io.vertx.proton.ProtonDelivery;
```

Comment:
please revert this change and adapt your IDE settings to the import order specified in `$HONO/eclipse/hono.importorder`

Reply:
Okay - found the option in the Eclipse Code Formatter plugin for IDEA..